Sources/PluginLibrary/swift-options.pb.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 import SwiftProtobuf
 
 
-let Google_Protobuf_FileOptions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
+let Extensions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
```

Comment:
`swiftFieldName` should probably match the property added to the struct, even if that means it starts with an uppercase letter (because of the package prefixing).

We have to choose between either (1) package-qualified property names that match the name mangling scheme, which means they'll start with an uppercase, or (2) introducing yet another name mangling scheme to make them start lowercase. I'd choose (1) just because we don't need yet another mapping. As long as those property names won't ever collide with generated type names, we should be ok.

No matter how we slice it, extensions are going to be ugly, so let's not twist ourselves too much about it.